app/containers/Preview.js

```diff
@@ -1,18 +1,12 @@
 import { connect } from 'react-redux';
 import PreviewComponent from '../components/Preview/Preview.js';
 
-const getEditorContent = (state) => {
-  if (state && state.form && state.form.editor && state.form.editor.values) {
+const mapStateToProps = (state) => {
+  try {
     return { ...state.form.editor.values };
+  } catch (_) {
```

Comment:
Yup, just stating that I'm not interested in using this variable. Would you prefer it renamed?